### PR TITLE
Add meta for _embedding_bag_dense_backward and _embedding_bag_per_sample_weights_backward

### PR DESCRIPTION
**Pull request by ezyang**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #125785

Signed-off-by: Edward Z. Yang <ezyang@meta.com>

### CODE REVIEW

File: test/test_meta.py [hunk 1]
```diff
@@ -1602,6 +1602,66 @@ def test_quantized_embedding_bag(self):
         self.assertEqual(eb.dtype, torch.float32)
         self.assertEqual(eb.untyped_storage().data_ptr(), 0)
 
+    # Tests mean and max.
+    # Can't easily test sum, because there is a fast path for sum which
+    # causes offset2bag to not get allocated... but the backward function
+    # needs it, and the offset2bag computation lives inside the
+    # derivatives.yaml formula directly, so there is no way to access it.
```
Comment (albanD) [inline test/test_meta.py:1609]: I'm not sure what you mean that the computation lives in the formula?

**Review by albanD — APPROVED** [review]
Sounds fair if CI is fixed

File: test/test_meta.py (hunk 1 above)
Comment (ezyang) [reply test/test_meta.py:1609]: you can have derivatives.yaml derivative do some compute, e.g., by directly calling a C++ function, before it eventually goes to an actual dispatched op

**Pull request closed without merging**